apps/sptribs/aat/00/kustomization.yaml

```diff
@@ -3,6 +3,6 @@ kind: Kustomization
 namespace: sptribs
 resources:
 - ../base
-- ../../sptribs-cron-migrate-case-flags/sptribs-cron-migrate-case-flags.yaml
```

Comment:
keep the cron base files in here. See example in: apps/nfdiv/aat/00/kustomization.yaml

Reply:
Initially we added here but still cron job is was not getting triggered. So we just moved to base file just to see if it creates the cron job in AAT.